continuedev/continue
# feat: configure docs through config.json

## Description

Configure docs through config.json

## Checklist


## Screenshots

[ "For visual changes, include screenshots." ]

## Testing

- Create a new doc through the UI
- Confirm it shows up in config.json
- Delete it through the UI
- Confirm it's removed
